cicecore/drivers/standalone/cice/CICE_RunMod.F90

```diff
@@ -160,7 +161,11 @@ subroutine ice_step
           biogeochemistry, save_init, step_dyn_wave, step_snow
       use ice_timers, only: ice_timer_start, ice_timer_stop, &
           timer_diags, timer_column, timer_thermo, timer_bound, &
-          timer_hist, timer_readwrite
+          timer_hist, timer_readwrite, &
+          timer_tmp1, timer_tmp2, timer_tmp3, timer_tmp4, timer_tmp5, timer_tmp6
+#ifdef OMP_TIMERS
+      use OMP_LIB
+#endif
```

Comment:
does it work with ` omp_lib` (I'm thinking it should) ? that would make case more consistent with our own modules.

Reply:
This code does not exist in this version.   I'm confused what's happening here.